Sources/SmileID/Classes/Camera/CameraManager.swift

```diff
@@ -179,3 +203,17 @@ extension CameraManager: AVCaptureVideoDataOutputSampleBufferDelegate {
         self.sampleBuffer = imageBuffer
     }
 }
+
+extension CameraManager: AVCapturePhotoCaptureDelegate {
+    func photoOutput(_ output: AVCapturePhotoOutput, didFinishProcessingPhoto photo: AVCapturePhoto, error: Error?) {
+        if let error {
+            return
```

Comment:
Does this mean if there's a photo capture error, that it will be silently swallowed? The error should be handled (and certainly logged)